### PR TITLE
fixes #147 add go to source btn

### LINKED ISSUE #147: Add "Go to Source" link in accordion view
In non-accordion view would it be possible to put a "Go to Source" view that links to the same link as source link on data set page? Like 

image

### PR DESCRIPTION
**Pull request by chloeleichen**
I didn't make it green as Allan suggested. I think it's better to be a bit more subtle than the view source button

### CODE REVIEW

File: magda-web-client/src/Dataset/DatasetSummary.js [hunk 1]
```diff
@@ -73,8 +73,11 @@ export default class DatasetSummary extends Component {
                     </span>
                   </div>
                   {this.props.isExpanded && <div className='dataset-summary__middle clearfix'>
-                      <div><Link className='btn dataset-summary__btn-view-dataset'
-                                 to={`/dataset/${encodeURIComponent(dataset.identifier)}`}>View dataset</Link></div>
+                      <div>
+                        <Link className='btn dataset-summary__btn-view-dataset'
+                                   to={`/dataset/${encodeURIComponent(dataset.identifier)}`}>View dataset</Link>
+                        <a className='btn dataset-summary__btn-go-to-source' href={dataset.landingPage}>Go to Source</a>
```
Comment (kring) [inline magda-web-client/src/Dataset/DatasetSummary.js]: What if the dataset doesn't have a landingPage property?

Comment (chloeleichen) [inline magda-web-client/src/Dataset/DatasetSummary.js]: Fixed

Comment (kring) [inline magda-web-client/src/Dataset/DatasetSummary.js]: Are you sure?  I don't see it.

Comment (chloeleichen) [inline magda-web-client/src/Dataset/DatasetSummary.js]: Sorry looks like i didn't commit that file. Fixed now

**Pull request merged**